build.gradle

```diff
@@ -1,8 +1,35 @@
 plugins {
-    id 'com.github.ben-manes.versions' version '0.39.0'
+  id 'com.github.ben-manes.versions' version '0.42.0'
 }
 
-allprojects {
+task archiveDeployment(type: Tar) {
+  destinationDirectory = file('.')
+  archiveFileName = 'deployment.tar'
+  into 'deployment'
+  from 'deployment'
+}
+
+// Add `distributeSql` task to all subprojects with `sql/` child directory
+configure(subprojects.findAll { it.projectDir.toPath().resolve('sql').toFile().exists() }) {
+  pluginManager.withPlugin('java') {
```

Comment:
The new Command Expansion Server is a node/javascript project. This project will have SQL schemas as well. Will this be overlooked as it is not java?

Reply:
Aha! You beat me to it 😂 
Mentioned this on your PR: https://github.com/NASA-AMMOS/aerie/pull/49#issuecomment-1043251309

Reply:
Basically just need to also consider the `com.github.node-gradle.node` plugin. I would preemptively add this but I'd rather add it to your PR, that way I can verify it works.

Reply:
I have another ticket [AERIE-1687](https://jira.jpl.nasa.gov/browse/AERIE-1687) we can work on instead of adding it to my current PR. The other ticket will utilize this PR.